Makefile

```diff
@@ -63,22 +81,16 @@ prere:
 	$(PREFIX)cd libft && make re
 
 clean:
-	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(VANIA_ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ)
-	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(VANIA_ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ)
+	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ) $(TEST_TOOL_OBJS)
+	$(PREFIX)@if [ -d $(OBJ_F) ]; then $(PREFIX)rm -rf $(OBJ_F); fi
 
 fclean: clean
-	$(PREFIX)rm -f $(NAME)
-	$(PREFIX)rm -f $(ENDPOINT_OBJ)
+	$(PREFIX)rm -f $(NAME) $(TEST_TOOL_FNAME)
```

Comment:
Not here

Reply:
propose where, i didnt want to add here testfclean, cause testclean will be called twice. but im ready to do as you say